README.md

```diff
@@ -2348,6 +2349,44 @@ Checks if string ends with the given target string.
 
 **[⬆ back to top](#quick-links)**
 
+### _.isString
+:heavy_exclamation_mark:`Not in Underscore.js`
+Checks if string ends with the given target string.
+
+  ```js
+  // Lodash
+  _.isString('abc');
+  // => true
+ 
+  _.isString(123);
+  // => false
+  
+  _.endsWith('abc', 'b', 2);
+  // => true
+
+  // Native
+  function isString(str){
+    if (str && typeof str.valueOf() === "string") {
```

Comment:
Please add test case :)

Reply:
ok sure

Reply:
@ODudek Added tests

Reply:
@ODudek also is it ok if i add two or three features in a single pr?

Reply:
I think you should do another PR for features :)

Reply:
i will be adding maybe 2-4 features on lang. Should i do all those in different prs or a single one?

Reply:
If you want to add native implementation of different methods you should create different prs ;)